src/components/registration/RegistrationForm.unit.test.tsx

```diff
@@ -98,6 +114,13 @@ describe("when text fields are submitted", () => {
         target: { value: TEST_LAST_NAME },
       }
     );
+
+    const parentDobInput = getByLabelText(
+      `${StudentRole.PARENT} ${DefaultFields.DATE_OF_BIRTH.name}`
+    ) as HTMLInputElement;
+    parentDobInput.setSelectionRange(0, parentDobInput.value.length);
+    userEvent.type(parentDobInput, TEST_PARENT_DOB);
```

Comment:
note: this'll be nice to extract to a util function since we're gonna do this in other tests, but we can leave it to whoever's writing the next one (prob me lol)

Reply:
this is a trick we have to do to test the datepicker, see https://stackoverflow.com/a/67715177